src/main/java/uk/gov/hmcts/sptribs/systemupdate/schedule/migration/task/SystemFinalOrderOverdueTask.java

```diff
@@ -0,0 +1,100 @@
+package uk.gov.hmcts.sptribs.systemupdate.schedule.migration.task;
+
+import lombok.extern.slf4j.Slf4j;
+import org.elasticsearch.index.query.BoolQueryBuilder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.sdk.type.YesOrNo;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.idam.client.models.User;
+import uk.gov.hmcts.sptribs.ciccase.model.State;
+import uk.gov.hmcts.sptribs.idam.IdamService;
+import uk.gov.hmcts.sptribs.systemupdate.service.CcdConflictException;
+import uk.gov.hmcts.sptribs.systemupdate.service.CcdManagementException;
+import uk.gov.hmcts.sptribs.systemupdate.service.CcdSearchCaseException;
+import uk.gov.hmcts.sptribs.systemupdate.service.CcdSearchService;
+import uk.gov.hmcts.sptribs.systemupdate.service.CcdUpdateService;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+import static org.elasticsearch.index.query.QueryBuilders.boolQuery;
+import static org.elasticsearch.index.query.QueryBuilders.matchQuery;
+import static org.elasticsearch.index.query.QueryBuilders.rangeQuery;
+import static uk.gov.hmcts.sptribs.systemupdate.event.SystemMigrateCase.SYSTEM_MIGRATE_CASE;
+
+@Component
+@Slf4j
+public class SystemFinalOrderOverdueTask implements Runnable {
+
+    @Autowired
+    private CcdSearchService ccdSearchService;
+
+    @Autowired
+    private CcdUpdateService ccdUpdateService;
+
+    @Autowired
+    private IdamService idamService;
+
+    @Autowired
+    private AuthTokenGenerator authTokenGenerator;
+
+    public static final String PRONOUNCED_DATE = "coGrantedDate";
+
+    private static final String FINAL_ORDER_OVERDUE_FLAG = "isFinalOrderOverdue";
+
+    @Override
+    public void run() {
+        log.info("Final Order overdue scheduled task started");
+
+        final User user = idamService.retrieveSystemUpdateUserDetails();
+        final String serviceAuth = authTokenGenerator.generate();
+
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+        String overdueDate = formatter.format(LocalDate.now().minusMonths(12));
+
+        try {
+            final BoolQueryBuilder query =
+                boolQuery()
+                    .must(boolQuery()
+                        .must(matchQuery("reference", 1688978122333564L))
+                    )
+
+                    /*.must(
+                        boolQuery()
+                            .should(boolQuery().must(rangeQuery(String.format(DATA, PRONOUNCED_DATE)).lt(overdueDate)))
```

Comment:
Creation Date would be much more suitable as you want to migrate all cases created before the code that is causing the issue was merged